sdk/nodejs/Makefile

```diff
@@ -24,6 +24,9 @@ ensure:: .make/ensure/yarn .make/ensure/node .make/ensure/phony
 	@mkdir -p .make/ensure && touch .make/ensure/phony
 
 format:: ensure
+	yarn biome format
+
+format_fix:: ensure
```

Comment:
Nit:

Ah, muscle memory... I'm personally kinda used to `make format` doing the formatting. Should we do the inverse of `lint` / `lint_fix` for `format`, e.g. `make format` does the formatting and something like `make format_check` or `make format_verify` does the checking?

I took a look at what we do in other repos, and it looks like `make format` does the formatting across the board: https://github.com/search?q=org%3Apulumi+path%3AMakefile+format&type=code&p=1.

The pulumi-service frontend does have `format` and `format:fix` scripts (as well as `lint` and `lint:fix`) in its `package.json`, but the `Makefile` has a `format` target that calls `format:fix`.

Is it better to be consistent with other repos or consistent with `lint` / `lint_fix`?

Reply:
I'll make it `format_check` :)

Reply:
I really wish we had consistent targets across our Makefiles!